app/dataflow/actions/types.js

```diff
@@ -1,4 +1,15 @@
-export const SET_CITY = 'SET_CITY';
-export const UPDATE_WEATHER_INFO = 'UPDATE_WEATHER_INFO';
-export const UPDATE_FORECAST = 'UPDATE_FORECAST';
-export const UPDATE_LOCATION = 'UPDATE_LOCATION';
+export default {
+    'SET_CITY': 'SET_CITY',
```

Comment:
@AlexeyBarkow why do you use string: string notation.

In common uses key: string. And one more suggestion o add a little more readability:

export default {
    LOAD: '[METEO] LOAD',
    LOAD_SUCCESS: '[METEO] LOAD SUCCESS',
    LOAD_FAIL: '[METEO] LOAD FAIL',
...
}

In this case you see key as name, and a bit more described actionType.
Also a good idea to separate actions by types: in your case as mininum weather and geo location.
So, next one:
GET_LOCATION: '[GEO] GET'
...